lib/mastodon/sidekiq_middleware.rb

```diff
@@ -2,9 +2,12 @@
 
 class Mastodon::SidekiqMiddleware
   BACKTRACE_LIMIT = 3
+  QUERY_LOG_TAGS_ENABLED = Rails.configuration.active_record.query_log_tags_enabled
```

Comment:
This works because the initializer is loaded after the config is set, but this might create a dependency issue. I am not sure how to properly work around it.

Reply:
Maybe just moving this out of a constant and in the method would be cleaner?

Reply:
I wondered about the cost of accessing this. Initially this was testing the environment variable directly so I put it into a constant. Will move to the method if you dont think it can be a problem.